src/components/forms/pool_actions/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { FullPool } from '@/services/balancer/subgraph/types';
+
+export function getPoolWeights(pool: FullPool) {
+  return Object.values(pool.onchain.tokens)
+    .map(token => `${token.weight * 100} ${token.symbol}`)
+    .join(', ');
+}
```

Comment:
This feels like the wrong place for this. Not sure where this type of logic should go though...

Maybe we could add a `/services/pool/pool.helper.ts` file for functions like these? Again, even that feels a bit like a hack. A question for later, should labels like these be part of the FullPool definition or should there be something else that provides that detail about a pool?

Reply:
Also, have you tested what this will look like on a 5-8 token pool? It might make the notification look over confusing.

Reply:
<img width="342" alt="Screenshot 2021-07-22 at 09 40 00" src="https://user-images.githubusercontent.com/2406506/126611785-c26e3afc-4b8d-40bc-80e6-83cf1f53b999.png">
<img width="346" alt="Screenshot 2021-07-22 at 09 40 07" src="https://user-images.githubusercontent.com/2406506/126611789-e72d798e-59b5-452c-93f7-393931f5459c.png">

Reply:
https://staging-kovan-app-git-evgeny-ui-669-tweak-notif-be487b-balancer.vercel.app/#/pool/0x702605f43471183158938c1a3e5f5a359d7b31ba000100000000000000000070

Reply:
All fixed now @garethfuller